win32ss/user/user32/misc/exticon.c

```diff
@@ -323,6 +323,42 @@ static UINT ICO_ExtractIconExW(
 	}
 	CloseHandle(fmapping);
 
+#ifdef __REACTOS__
+    /* Check if the resource is an animated icon/cursor */
+    if (!memcmp(peimage, "RIFF", 4))
+    {
+        INT     anihOffset;
+        ULONG   uSize = 0;
+
+        /* Get size of the animation data */
+        uSize = peimage[4] + (peimage[5] << 8);
```

Comment:
For the shift you probably want to cast, or also use `MAKEWORD` like below

Reply:
OK. Changed to MAKEWORD. Thanks.